thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DashboardResource.java

```diff
@@ -558,4 +565,29 @@ public String getData(@QueryParam("type") String type) throws Exception {
     }
     return response;
   }
+
+  @POST
+  @Path("/severity")
+  public Response computeSeverity(@NotNull @QueryParam("collection") String collectionName,
+      @NotNull @QueryParam("metric") String metricName,
+      @NotNull @QueryParam("start") String startTimeIso, @NotNull @QueryParam("end") String endTimeIso)
+      throws Exception {
+    DateTime startTime = null;
+    DateTime endTime = null;
+    if (StringUtils.isNotBlank(startTimeIso)) {
+      startTime = ISODateTimeFormat.dateTimeParser().parseDateTime(startTimeIso);
+    }
+    if (StringUtils.isNotBlank(endTimeIso)) {
+      endTime = ISODateTimeFormat.dateTimeParser().parseDateTime(endTimeIso);
+    }
+    PinotThirdEyeClient thirdEyeClient = PinotThirdEyeClient.getDefaultTestClient();
+    SeverityComputationUtil util = new SeverityComputationUtil(thirdEyeClient, collectionName, metricName);
+    long currentWindowStart = startTime.getMillis();
+    long currentWindowEnd = endTime.getMillis();
+    String compareMode = "WO3WMean";
```

Comment:
add a TODO to support more compare Mode. Also with the new change to sign test algo, we might want wo4wMean right?